eos/saveddata/fit.py

```diff
@@ -1257,16 +1256,16 @@ def fits(self):
 
         return True
 
-    def __deepcopy__(self, memo):
-        copy = Fit()
+    def __deepcopy__(self, memo=None):
+        copy_ship = Fit()
         # Character and owner are not copied
-        copy.character = self.__character
-        copy.owner = self.owner
-        copy.ship = deepcopy(self.ship, memo)
```

Comment:
Fair enough. Looking it up it seems to keep a ledger of the original objects that can be accessed outside of the deepcopy. We don't use it, so this is probably fine.